source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -1388,6 +1398,16 @@ public bool IsTaxiLightOn
 			set => Function.Call(Hash.SET_TAXI_LIGHTS, Handle, value);
 		}
 
+		public VehicleIndicatorLight IndicatorLight
+		{
+			get => (VehicleIndicatorLight)NativeMemory.ReadByte(MemoryAddress + NativeMemory.IsInteriorLightOnOffset);
```

Comment:
`CVehicle` uses all the bits at the offset of `IsInteriorLightOnOffset` (0x949 in v1.0.2699.0) according to the dumped exe. As I tested, the 5th to 8th bits aren't relevant to the indicator lights at least.

Reply:
Thanks, now i just check the first two bits, `VehicleIndicatorLight` seems to be unnecessarry then.